addon/utils/clamp.js

```diff
@@ -84,6 +75,12 @@ export default (function(win, doc) {
       }
     }
 
+    // measurement element is made a child of the clamped element to get it's style
+    measure = ce('span');
+    measure.style.position = 'absolute'; // prevent page reflow
+    measure.style.whiteSpace = 'pre'; // cross-browser width results
+    measure.style.visibility = 'hidden'; // prevent drawingg
```

Comment:
s/drawingg/drawing

Also, can you please move this nearer the initialization section above?